openedx/core/djangoapps/user_authn/views/tests/test_password.py

```diff
@@ -176,6 +177,36 @@ def test_password_change(self):
         response_dict = json.loads(response.content.decode('utf-8'))
         assert response_dict['success']
 
+    @ddt.data(
+        (True, True, 'walter@graymattertech.com'),
+        (True, False, 'walter@graymattertech.com'),
+        (False, True, 'walter@graymattertech.com'),
+        (False, False, 'edx@example.com'),
```

Comment:
4th case should fail because in that case, email should be taken, not email_from_support_tools

Reply:
This is what actually happens. It just sends password reset email to the logged in user. That's why I'm using `'edx@example.com'`